src/main/java/fr/inria/diversify/mutant/pit/GradlePitTaskAndOptions.java

```diff
@@ -0,0 +1,58 @@
+package fr.inria.diversify.mutant.pit;
+
+/**
+ * Created by Daniele Gagliardi
+ * daniele.gagliardi@eng.it
+ * on 21/07/17.
+ */
+
+public class GradlePitTaskAndOptions {
+
+    public static boolean descartesMode = false;
+
+    public static boolean evosuiteMode = false;
+
+    public static final String OPT_WITH_HISTORY = "enableDefaultIncrementalAnalysis = ";
+
+    public static final String OPT_VALUE_REPORT_DIR = "reportDir = 'build/pit-reports'";
+
+    public static final String OPT_TARGET_CLASSES = "targetClasses = ";
+
+    public static final String OPT_VALUE_FORMAT = "outputFormats = ['CSV','HTML']";
+
+    public static final String OPT_TARGET_TESTS = "targetTests = ";
+
+    public static final String PROPERTY_ADDITIONAL_CP_ELEMENTS = "additionalMutableCodePaths = ";
+
+    public static final String PROPERTY_EXCLUDED_CLASSES = "excludedClasses = ";
+
+    //Apparently Unsupported by PIT plugin
+    public static final String OPT_ADDITIONAL_CP_ELEMENTS = "classPath = ";
```

Comment:
It seems that their documentation is not really consistent.

I think we should use the properties file rather than the command line because DSpot is already using it for the additionnalClassPathElements and excludedClasses. It seems legit to do the same for those two options: jvmArgs and timeConstInMillis.
Thank you.